docs/tools/CLI/cli-setup/cli-reqs.md

```diff
@@ -228,6 +228,12 @@ Mbed CLI supports a setting for each toolchain path:
 | IAR EWARM Compiler | `C:/Program Files/IAR Systems/Embedded Workbench 8.2/arm/bin/iccarm.exe` | `IAR_PATH` | `C:/Program Files/IAR Systems/Embedded Workbench 8.2/arm`|
 | GCC Arm Embedded Compiler | `/usr/bin/arm-none-eabi-gcc` | `GCC_ARM_PATH` | `/usr/bin`|
 
+##### Using Arm Compiler 6 from Mbed Studio
+Mbed Studio comes with a free version of Arm Compiler 6 when it's used with Mbed OS. It requires configuring the `ARMC6_PATH` as well as setting an environment variable to manage the license.
+
+For Mbed Studio version 0.4.0, the Arm Compiler 6 executable is located in `./tools/ac6/bin`. This path is relative to the installation directory of Mbed Studio. On Windows for example, the default path is `C:\MbedStudio\tools\ac6\bin`.
```

Comment:
They have a new release next week - 0.5

Reply:
Good to know, hopefully they can comment if I need to update this.

Reply:
Please remove this if you accept the suggestion above

Reply:
@bridadan Irit is correct. Version 0.5 of Mbed Studio is planned soon. Ideally, you should not add path in docs. We are also moving the location of tools. It is no longer going to be inside of installation path.

Reply:
Arek also said it's going to be different locations for different platforms. 

This is starting to look like the docs maintenance effort is going to be very high.